Utils.js

```diff
@@ -96,7 +96,7 @@ function configToJson(config) {
         }
         // APM END --------------------------------------------
         if (config.disableAdditionalIntentRedirectionChecks) {
-            json['disableAdditionalIntentRedirectionChecks'] = config.disableAdditionalIntentRedirectionChecks;
+            json.disableAdditionalIntentRedirectionChecks = config.disableAdditionalIntentRedirectionChecks;
```

Comment:
maybe you can return this back. dunno why prettier jumped on to this

Reply:
done